### ISSUE TITLE
[Alfresco] Lose of secondary parent

### ISSUE THREAD

**Issue by asauvez** [aegif/CmisSync#726]
In Alfresco 5.1, I have a node "common_folder" in Alfresco with a primary parent (Site_1) and a secondary parent (Site_2):
/Site_1/documentLibrary/common_folder/foo/bar.docx
and 
/Site_2/common_folder/foo/bar.docx

When I synchronize with CmisSync the folder /Site_2/ with a local folder, CmisSync change the primary parent of common_folder to Site_2. The link between Site_1 and common_folder is lost.

Is there any way arround this behavior ?

This issue may be related to #533.

**Comment by asauvez** [comment]
Yes, as soon as the sync is done, even if I don't change anything on the Alfresco side or the local folder.

With debugging, I can see that CmisSync delete all the nodes bellow the secondary children association, then recreate them bellow Site_2. In my example, foo and bar.docx doesn't have the same nodeRef before and after the sync.

Thant for your time and your very usefull tool.

**Comment by nicolas-raoul** [comment]
About the multiple parents idea itself: #533
